source/air/src/ast.rs

```diff
@@ -48,6 +50,21 @@ pub enum BinaryOp {
     Gt,
     EuclideanDiv,
     EuclideanMod,
+
+    BitXor,
+    BitAnd,
+    BitOr,
+    BitAdd,
+    BitSub,
+    BitMul,
+    BitDiv, // unsigned
+    BitLt,  // unsigned
+    BitGt,  // unsigned
+    BitLe,  // unsigned
+    BitGe,  // unsigned
+    BitMod, // unsigned
+    Shr,    // logical
+    Shl,
```

Comment:
could the signedness be incorporated into the name? e.g., LShr, or UDiv?

Reply:
Sure, that sounds good